sdk/identity/azure_identity/Cargo.toml

```diff
@@ -26,9 +26,6 @@ openssl = { workspace = true, optional = true }
 pin-project.workspace = true
 typespec_client_core = { workspace = true, features = ["derive"] }
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-async-process.workspace = true
-
```

Comment:
This should be removed from the workspace if it's the only one referencing it left.